prod-cockpit-rcar.yaml

```diff
@@ -595,3 +597,38 @@ parameters:
               env:
                 - "DDK_UM_PREBUILDS=eva/pvr-um"
 
+  ENABLE_AOS_VIS:
```

Comment:
`ENABLE_AOS_VIS` option doesn't look so suitable here as it enables whole Aos core. Consider to rename it to 'ENABLE_AOS`.